src/sequelize/migrations/20190620145553-add-readtime.js

```diff
@@ -1,10 +0,0 @@
-
```

Comment:
Pair with @eliemugenzi to ensure `Add read time` feature works without any glitches.